llvm/include/llvm/TableGen/SearchableTable.td

```diff
@@ -135,6 +138,12 @@ class SearchIndex {
   //
   // Can only be used when the first field is an integral (non-string) type.
   bit EarlyOut = false;
+
```

Comment:
Should we remove this since it always generate an error?

Reply:
The primarykey field in genericTable is an instance of `SearchIndex` and is parsed using `parseSearchIndex` which is also used for parsing non-primary functions. Therefore it is necessary to keep `ReturnRange` in `SearchIndex` even though it will always throw an error.